Grid.elm

```diff
@@ -10,22 +11,15 @@ import Matrix.Extra
 -- Battleship
 import Ship
 
-
 -- Grid
 type alias IsHit = Bool
 type alias Grid = Matrix.Matrix Cell
 type Cell
     = Ship IsHit
     | Empty IsHit
+    -- TODO: Sunk state
```

Comment:
See? That's a good example why you should focus on the UI first — your imagination won't be enough to think of all the possibilities. Also, while we're on it, there doesn't need to be four fields in the Play view — two will suffice.